src/spikeinterface/core/base.py

```diff
@@ -102,7 +104,7 @@ def ids_to_indices(self, ids: list | np.ndarray | None = None, prefer_slice: boo
 
         Parameters
         ----------
-        ids : list or np.ndarray
+        ids : list or np.ndarray or tuple or None
```

Comment:
```suggestion
        ids : list | np.ndarray | tuple | None, default: None
```

Just so we have the docstring in full compliance. The pipes aren't crucial but we need to add in the default value.